scene/gui/text_edit.cpp

```diff
@@ -3926,6 +3926,10 @@ void TextEdit::_update_completion_candidates() {
 	int ci_match=0;
 	for(int i=0;i<completion_strings.size();i++) {
 		if (completion_strings[i].begins_with(s)) {
+			// don't remove duplicates if no input is provided
```

Comment:
Was about to ask "why?", but then I checked the autocompletion of ``event.`` in an ``_input(event)`` callback, and indeed that makes sense :)